pkg/bootstrapper/bootstrapper.go

```diff
@@ -594,7 +597,7 @@ func (wmcb *winNodeBootstrapper) ensureKubeletService() error {
 		ServiceStartName: "",
 		DisplayName:      "",
 		Password:         "",
-		Description:      "OpenShift Kubelet",
+		Description:      managedString,
```

Comment:
Any reason not to just add "openshift-managed" to the existing service description?

Reply:
The description was arbitrarily added when writing WMCB
it would look weird if it was changed to `OpenShift Kubelet openshift-managed`

Especially since the service name already calls it `kubelet`

Reply:
I'm also thinking in general, for other services with more informative descriptions, maybe prepending `managedString`? So here we could fix up kubelet description e.g. 
```
"openshift-managed: registers an instance as a node with the cluster's api-server"
```

Reply:
I suggest something like `OpenShift managed Kubelet`.